index.html

```diff
@@ -25,13 +25,16 @@
     <header class="hamburger">
       <div class="head">
         <div class="name-me">Yohannes Zemed</div>
-        <img class="mob-img" src="icons/menu.png" alt="menu" />
+        <img id="btn" class="mob-img" src="icons/menu.png" alt="menu" />
         <div class="desk-head">
           <ul class="desk-li">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contacts</li>
-            <li>
+            <li class="close">
+              <img id="close" src="icons/Icon.png" alt="" />
+            </li>
+            <li class="common-li" id="portfolio">Portfolio</li>
+            <li class="common-li" id="about">About</li>
+            <li class="common-li" id="contact">Contacts</li>
```

Comment:
- [ ] **Required:** Excellent job implementing this 👏. To ensure it is completely similar  to your chosen template, please add the following:

-   Change the `font-family` from Times new roman to Inter, popkin etc as used in your project.
- Align the text to the left as shown in the Figma Template also attached below.
- Add some padding to the right to push the `hr` away from the screen.

Your Implementation | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150930445-ec1fd612-6c57-4ffc-ae98-28efa398e4d5.png) | ![image](https://user-images.githubusercontent.com/27709832/150930585-606832ea-c997-4d60-ab64-e86a71586476.png)

Reply:
Thanks for all the comments and suggestions.